bool subscribe
string device
uint16 width
uint16 height
uint8 fps
uint16 camBufferSize
uint8 compressionRatio
string calibrationFile
---
int8 ret